share/functions/export.fish

```diff
@@ -10,7 +10,7 @@ function export --description 'Set global variable. Alias for set -gx, made for
                 set -gx $v $$v
             case 2
                 if contains $v[1] PATH CDPATH MANPATH
-                    set -gx $v[1] (string split ":" $v[2])
+                    set -gx $v[1] (string split ":" (string replace " " ":" $v[2]))
```

Comment:
Why?

Maybe I'm misremembering what other shells do here, but I was under the impression that a $PATH set to `foo bar:baz` would be interpreted to have two components - "foo bar" and "baz".